FRPs/FRP-15.md

```diff
@@ -0,0 +1,27 @@
+---
+id: 15
+title: MEV in eth2 - further analysis
+team: @obadiaa
+created: 2021-06-04
+status: pending
+---
+
+# MEV in eth2 further analysis
```

Comment:
this one is a bit vague still, I'd put some more motivation for the questions in the intro section--why is more analysis needed? why these particular metrics?